src/main/resources/templates/c_plus_plus/test/testUtils/AbstractTest.py

```
@@ -0,0 +1,274 @@
+        """
+        outDir: str = self._getOutputPath()
+        if path.exists(outDir) and path.isdir(outDir):
+            return
+        makedirs(outDir)
+
+    def _getOutputPath(self):
+        """
+        Returns the output path for temporary stuff like the stderr and stdout files.
+        """
+
+        return path.join("/tmp", self.suite.name, self.name)
```

Ruff

Probable insecure usage of temporary file or directory: "/tmp"

(S108)